appveyor.yml

```diff
@@ -51,6 +51,7 @@ install:
 build: false  # Not a C# project, build stuff at the test step instead.
 
 before_test:
+  - "%CMD_IN_ENV% pip install psycopg2==2.6.2"
```

Comment:
This installs psycopg2 from a wheel, which links it differently from doing a pip install. I'm not 100% certain tbh but I know it's necessary.